rate: 5 # control loop frequency [Hz]
parent_frame: map # TODO: set correct value
child_frame: ego_vehicle # TODO: set correct value
max_iterations: 2000 # maximum interations per control loop
gamma: 40 # RRT*
dist_threshold: 5 # TODO: tune this
angle_threshold: 0.3490658503988659 # TODO: tune this
step_size: 10.0 # TODO: tune this
goal_bias: 0.3 # proability that the goal is returned by sampler
